utils/src/treev2/quickPickWizard/QuickPickWizardContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../index';
+
+export function getLastNode<TNode = unknown>(context: types.QuickPickWizardContext): TNode | undefined {
+    if (context.pickedNodes.length) {

```

Comment:
Maybe: `return context.pickedNodes.at(-1) as TNode;` ?

Reply:
Nice, forgot we could use that now. Note: had to update `@types/node` to v16.